src/test/java/org/llorllale/cactoos/matchers/EndsWithTest.java

```diff
@@ -50,7 +50,7 @@ public void matches() {
     public void mismatches() {
         new Assertion<>(
             "The matcher gives negative result for the invalid arguments",
-            new EndsWith("!"),
+            new EndsWith("The sentence!"),
```

Comment:
@ruca905 can you just revert it to the original code? It was correct before I think...